src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -202,6 +212,19 @@ public Fluid setEmptySound(SoundEvent emptySound)
         return this;
     }
 
+    /**
+     *  The color parameter includes alpha! so ARGB format
+     *  For example, if you want pure green (which is RGB #00FF00)
+     *  you probably want to pass 0xFF00FF00 so there is no
+     *  alpha transparency. It also is blended with color
+     *  of your texture, so if you want to preserve texture
+     *  then leave at default value (white).
+     */
+    public Fluid setColor(int color) 
```

Comment:
It may be simpler if you take a `java.awt.Color` parameter, that way there is no question about the format.

Reply:
I actually wanted to do that but the getter was an int so I felt there should be a symmetrical method. However, I'm quite happy to add a second setter method that takes Color. I'll do that.

Reply:
I don't think we need to keep this method, having just one with  `Color` should be good since it doesn't need the long explanation.